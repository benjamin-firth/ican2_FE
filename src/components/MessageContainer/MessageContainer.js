import React from 'react';
import './MessageContainer.scss';
import Message from '../Message/Message';
import MessageForm from '../MessageForm/MessageForm';

const MessageContainer = () => {

  return (
    <section className='message-container'>
      <p>This is a message container</p>
    </section>
  );
}

export default MessageContainer;
