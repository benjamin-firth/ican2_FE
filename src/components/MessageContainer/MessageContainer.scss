@import '../../variables.scss';

.message-container {
  padding-top: 12vw;
  height: 100vw;
  background-color: $primarycolor;
  display: flex;
  flex-direction: column;
  justify-content: center;
}
