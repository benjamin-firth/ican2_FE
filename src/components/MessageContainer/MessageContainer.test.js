import React from 'react';
import { shallow } from 'enzyme';
import MessageContainer from './MessageContainer';

describe('MessageContainer', () => {
  let wrapper;
  let mockOther = { name: 'mock name' };

  beforeEach(() => {
    wrapper = shallow( <MessageContainer other={mockOther} />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});