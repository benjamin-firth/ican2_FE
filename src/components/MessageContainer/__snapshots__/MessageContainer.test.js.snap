// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageContainer should match the snapshot 1`] = `
<section
  className="message-container"
>
  <p>
    This is a message container between you and 
    mock name
  </p>
  <Message />
  <Message />
  <MessageForm />
</section>
`;
